src/envoy/http/mixer/filter.cc

```diff
@@ -221,6 +221,8 @@ void Filter::completeCheck(const CheckResponseInfo& info) {
     int status_code = ::istio::utils::StatusHttpCode(status.error_code());
     decoder_callbacks_->sendLocalReply(Code(status_code), status.ToString(),
                                        nullptr);
+    decoder_callbacks_->requestInfo().setResponseFlag(
+        RequestInfo::ResponseFlag::UnauthorizedExternalService);
```

Comment:
Is suppose `fail policy` is used somewhere in `istio::utils::StatusHttpCode` or `sendLocalReply` ?

Reply:
It's in gRPC transport used by mixerfilter. Fail-open means you will get status ok here.

Reply:
thnx, can you update the description to `Set ResponseFlag`